src/components/general/equipment/Equipment.jsx

```diff
@@ -406,6 +406,28 @@ export const Equipment = () => {
                     abortController.abort();
                 };
             }
+            
+            if(state.openedTab) {
+                let selectedTab;
+                switch (state.openedTab) {
+                case 'Inventory':
```

Comment:
Missing indent.

Reply:
Got it. Will keep an eye on that next time.